### PR TITLE
1.3

### PR DESCRIPTION
**Pull request by mensfeld**
Ready to be merged.

### CODE REVIEW

File: spec/lib/karafka/serialization/json/serializer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  describe '.call' do
+    context 'when content is a string' do
+      let(:content) { rand.to_s }
+
+      it 'expect not to do anything' do
+        expect(serializer.call(content)).to eq content
+      end
+    end
+
+    context 'when content can be serialied with #to_json' do
```
Comment (benissimo) [inline spec/lib/karafka/serialization/json/serializer_spec.rb]: `serialied` -> `serialized` here and below on line 23

Comment (mensfeld) [reply spec/lib/karafka/serialization/json/serializer_spec.rb]: Thanks, fixed.

**Comment by benissimo** [comment]
Looking forward to this getting merged into master!

**Comment by mensfeld** [comment]
@benissimo sorry it takes so much time. Release notes and stuff :/ @nijikon will do a final review this week and next week I will craft a beta release.

**Pull request merged**